test/test_jax_interop.py

```diff
@@ -25,6 +24,67 @@ def f(a, b):
     torch.testing.assert_close(
         b, torch.sin(torch.ones(3, 3)) + 1, check_device=False)
 
+  def test_call_jax_grad(self):
+    """
+    Test that we can call a JAX function and retrive grad from PyTorch/XLA lazy
+    tensor tracing.
+    """
+    dev = xm.xla_device()
+    a = torch.randn((3, 3), device=dev, requires_grad=True)
+    b = torch.randn((3, 3), device=dev, requires_grad=True)
+
+    ##############pytorch function##############
+    def f_torch(a, b):
+      return torch.sum(torch.cos(a) + torch.sin(b))
+
+    def f_backward_torch(f, a, b):
+      out = f(a, b)
+      print(out)
+      out.backward()
+      return a.grad, b.grad
+
+    _a = a.clone().requires_grad_(True)
```

Comment:
What's the purpose of cloning from `a` to `_a`? Usually variable names like `_a` is reserved for variables that are never read. Also I wonder if we could avoid the cloning and just use `a`. If you wanted to avoid accumulating gradient into `a`, then the right incantation is `a.clone().detach().requires_grad_(True)`.